themes/default/content/docs/guides/continuous-delivery/github-actions.md

```diff
@@ -397,10 +397,10 @@ any resource changes will be left on the Pull Request, as well as links to the P
 Console for more detailed information.
 
 You can install the Pulumi GitHub App now, by visiting
-[github.com/apps/pulumi](https://github.com/apps/pulumi) or clicking the button below.
```

Comment:
Actually just noticed this -- and I think I get why you're removing it ('cause directional reference), but do you think it's strange to say "install the GitHub app by going here" and then placing a button right below that that reads "Install"? I'm thinking people may be slightly confused by this (since "click the button below", directionality notwithstanding, would've at least explained that).

![image](https://user-images.githubusercontent.com/274700/142330953-55da66c4-c8ab-42c1-a914-c08bb27f5453.png)

Reply:
@cnunciato heh i actually had originally updated the button text to be more specific, e.g, Install Pulumi GitHub App. would you support that instead? but then i backed out.. i can do that though.

and if i do that, would you support removing the you can.. language, with direct link?

Reply:
Yep, I was just about to suggest something just like that. Sounds perfect. 👍